sdk/sdk/src/test/java/org/apache/carbondata/sdk/file/AvroCarbonWriterTest.java

```diff
@@ -603,4 +605,161 @@ public void testWriteBasicForFloat() throws IOException {
     }
   }
 
+  @Test
+  public void testAvroFileLoadWithPrimitiveSchema() throws IOException {
```

Comment:
This testcase is redundant. It is covered in testLoadingAvroFileAndReadingCarbonFile

Reply:
Removed this test case and also refactored all the testcases for all the files.